src/main/java/com/cloudbees/jenkins/support/filter/DataFaker.java

```diff
@@ -52,14 +52,14 @@ public static DataFaker get() {
         return ExtensionList.lookupSingleton(DataFaker.class);
     }
 
-    private final Faker faker = new Faker(Locale.ENGLISH);
+    private final RandomNameGenerator faker = new RandomNameGenerator();
```

Comment:
A little picky, but the fact that this variable is still called faker was a little confusing to me at the beginning.